core/templates/pages/topics-and-skills-dashboard-page/modals/delete-skill-modal.component.ts

```diff
@@ -31,23 +42,75 @@ export class DeleteSkillModalComponent extends ConfirmOrCancelModal {
   // and we need to do non-null assertion. For more information, see
   // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
   skillId!: string;
+  nonEligibleTopicNameToTopicAssignments !: TopicNameToTopicAssignments;
```

Comment:
Drop the space before the !, for consistency?

Reply:
Updated